docs/skopeo.1.md

```diff
@@ -123,6 +123,13 @@ Print the version number
   Default directory containing registry configuration, if **--registries.d** is not specified.
   The contents of this directory are documented in [containers-registries.d(5)](https://github.com/containers/image/blob/main/docs/containers-registries.d.5.md).
 
+## EXIT STATUS
```

Comment:
Non-blocking: `man-pages(7)` says the `EXIT STATUS` section should be before `FILES`.

Reply:
Thanks, I fixed this too, I love this level of attention to detail, really appreciated.